bigbluebutton-html5/imports/ui/components/join-handler/presenceManager/component.tsx

```diff
@@ -1,8 +1,5 @@
 import { useMutation, useQuery, useSubscription } from '@apollo/client';
-import React, { useContext, useEffect, useRef } from 'react';
-import { Meteor } from 'meteor/meteor';
-// @ts-ignore - type avaible only to server package
-import { DDP } from 'meteor/ddp-client';
+import React, { useContext, useEffect } from 'react';
```

Comment:
We can likely remove https://github.com/bigbluebutton/bigbluebutton/blob/v3.0.x-release/bigbluebutton-html5/.meteor/packages#L23 now too.